src/index.html

```diff
@@ -6,9 +6,58 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="logo">
+        <a href="#">
+          <img src="/src/images/logo.png" alt="MOYO logo">
+        </a>
+      </div>
+
+      <nav>
+        <ul>
+          <a href="#" class="is-active">
+            <li>Apple</li>
+          </a>
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item>
            <a href="#" class="nav__link is-active">
              Apple
            </a>
          </li>
```

link should be in `li`.
add classes for each element and style by classes instead by tags.
suggest such class naming